### PR TITLE
Revert dot to hyphen references in OPP

### PR DESCRIPTION
**Pull request by dtaylor7**
# Description

[DIV-3794 - OPP - AddAnother is broken from our 4.X.X release](https://tools.hmcts.net/jira/browse/DIV-3794)

This PR reverts a change made a while ago to swap dots for hyphens for input names. This was due to a bug in GOVUK elements which has now been resolved.

The reason we are reverting is because the previous change from dots to hyphens broke the Add Another functionality of One Per Page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,17 +28,18 @@
...
-    "@hmcts/one-per-page": "https://github.com/hmcts/one-per-page.git#4.1.4",
+    "@hmcts/one-per-page": "^5.0.0",
     "applicationinsights": "^1.0.3",
     "cookie-parser": "^1.4.3",
     "cookies": "^0.7.1",
     "copy-webpack-plugin": "^4.5.2",
     "express": "^4.16.3",
     "express-limiter": "^1.6.1",
     "glob": "^7.1.2",
+    "hard-source-webpack-plugin": "^0.13.1",
```
Comment (hosainnet) [inline package.json]: Sorry what is this for?

Comment (dtaylor7) [reply package.json]: a laugh...

Comment (dtaylor7) [reply package.json]: yea we dont need this....

Comment (dtaylor7) [reply package.json]: ill open new pr to remove

Comment (hosainnet) [reply package.json]: Oh okay - if you elaborated initlally maybe Vivek wouldn't have merged 😅

Comment (dtaylor7) [reply package.json]: i was refreshing my memory at the time...

**Pull request merged**